docs: add autoscaler ACL policy requirements. (#20041)
hashicorp/nomad: website/content/tools/autoscaling/agent.mdx

}
```

> When utilizing the High-Availability feature added in v0.4.0, the ACL policy will
> need to include capabilities that allow it to obtain and release Nomad variable

Reviewer: ```suggestion
When utilizing the High-Availability feature added in v0.4.0, the ACL policy
needs to include capabilities that allow the Nomad Autoscaler to obtain and release Nomad variable
```

locks. The namespace identifier should match the configured
[`high_availability.lock_namespace`][autoscaler_ha_lock_namespace] parameter,
and the path should match the [`high_availability.lock_path`][autoscaler_ha_lock_path]
parameter.

```hcl
namespace "default" {
  path "nomad-autoscaler/lock" {
    capabilities = ["write"]
  }
}
```

> If running Nomad Autoscaler Enterprise, the following ACL policy addition will be

Reviewer: ```suggestion
If running Nomad Autoscaler Enterprise, the following ACL policy addition is
```
